Index: src/defines.h
===================================================================
--- src/defines.h
+++ src/defines.h	2012-01-31 13:15:02.829012845 +0100
@@ -391,7 +391,7 @@
 #ifdef LINUX
 #define INI_USER_DIRECTORY "~/.berusky"
 #define INI_USER_LEVELS    "~/.berusky/User"
-#define INI_FILE_GLOBAL    "/var/games/berusky/"INI_FILE_NAME
+#define INI_FILE_GLOBAL    "/etc/berusky/"INI_FILE_NAME
 #define INI_FILE_USER      "~/.berusky/"INI_FILE_NAME
 #define INI_FILE_LOCAL     "./"INI_FILE_NAME
 #elif WINDOWS
Index: data/berusky.ini
===================================================================
--- data/berusky.ini
+++ data/berusky.ini	2012-01-31 13:32:36.136844683 +0100
@@ -1,9 +1,9 @@
 # Configuration for berusky game
 
 # Game data location
-level_data = /usr/share/berusky/Levels
-game_data = /usr/share/berusky/GameData
-graphics_data = /usr/share/berusky/Graphics
+level_data = /usr/share/games/berusky/Levels
+game_data = /usr/share/games/berusky/GameData
+graphics_data = /usr/share/games/berusky/Graphics
 
 # Graphics settings for game and editor
 # game runs in 640x480
@@ -17,8 +17,8 @@
 
 # Game binary & tmp dir
 # they're used by editor for "run level" command
-game_binary = /usr/bin/berusky
-tmp_data = /var/tmp
+game_binary = /usr/games/berusky
+tmp_data = /tmp
 
 # Dir for users levels (it's the default level dir for level editor)
 level_data_user = ~/.berusky/User
